app/ui-react/packages/ui/src/Shared/HelpDropdown.tsx

```diff
@@ -13,6 +13,11 @@ export interface IHelpDropdownProps {
   className?: string;
   isOpen: boolean;
   launchAboutModal: any;
+  launchSupportPage: any;
```

Comment:
If I were to be nitpicky I'd say that instead of any you should have written`(event: React.MouseEvent<HTMLAnchorElement>) => void | false`, where `React.MouseEvent` is the generic type that React uses for mouse related events - a click in this case, `HTMLAnchorElement` is [the underlying type of the `DropdownItem`](https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.d.ts#L3), and the return `void | false` is to describe the allowed return values accordingly to the [event handler processing algorithm](https://html.spec.whatwg.org/multipage/webappapis.html#the-event-handler-processing-algorithm)  for this kind of event.

But this would be overkill. I'd simply write it as a `() => void` since that's what our component expects to handler to do.